### PR TITLE
feat(connectAnotherDevice): 25% connectAnotherDevice rollout.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Remove connectAnotherDevice from the `verification` experiments. Place
50% of users into the `connectAnotherDevice` experiment, ignoring `isMetricsEnabledValue`. 25% of users will go into each of the two buckets, compared to 1.25% now.

@vladikoff - r?

We'll want this on the train-79 tag.

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
After looking at this content server code, I realized this alone does not fully do what I need it to do. There will be a content server component to this as well.

File: connect-another-device/connect-another-device.js [hunk 1]
```diff
@@ -4,15 +4,25 @@ module.exports = {
   name: 'Should the user see the "connect another device" screen',
   hypothesis: 'A nudge to connect another device will help increase multi-device users',
   startDate: '2015-01-01',
-  subjectAttributes: ['uniqueUserId', 'isMetricsEnabledValue', 'forceExperimentGroup'],
+  subjectAttributes: ['uniqueUserId', 'forceExperiment', 'forceExperimentGroup'],
   independentVariables: ['connectAnotherDevice'],
```
Comment (vladikoff) [inline connect-another-device/connect-another-device.js:8]: @shane-tomlinson could you also add `false` for this value into https://github.com/mozilla/fxa-content-experiments/blob/dev/defaults.json

File: connect-another-device/connect-another-device.js [hunk 2]
```diff
@@ -4,15 +4,25 @@ module.exports = {
...
+    // Place 50% of users into the two buckets, meaning the feature
+    // will be enabled for ~25% of users. 'isMetricsEnabledValue' is
+    // ignored, we have sufficiently high numbers that we'll assume
+    // an equal number of users who have metrics enabled are in each
+    // bucket. This assumption is measured in DataDog.
+    return this.bernoulliTrial(0.5, subject.uniqueUserId);
   },
+
   groupingFunction: function (subject) {
     var GROUPS = ['control', 'treatment'];
```
Comment (vladikoff) [inline connect-another-device/connect-another-device.js:27]: @shane-tomlinson there won't be groups anymore AFAIK

Comment (shane-tomlinson) [reply connect-another-device/connect-another-device.js:27]: Yeah, still groups, I still want to run this as an AB test, Where A is the existing flow, B is the new flow.

File: connect-another-device/connect-another-device.js (hunk 1 above)
Comment (shane-tomlinson) [reply connect-another-device/connect-another-device.js:8]: Done.

**Comment by shane-tomlinson** [comment]
I'm going to rebase this off of #53 once that merges.

**Comment by vladikoff** [comment]
Please rebase...

**Pull request merged**